cypress-studio/cypress/integration/checkout/shopping-cart.spec.js

```diff
@@ -0,0 +1,36 @@
+// shopping-cart.spec.js created with Cypress
+//
+// Start writing your Cypress tests below!
+// If you're unfamiliar with how Cypress works,
+// check out the link below and learn how to write your first test:
+// https://on.cypress.io/writing-first-test
+/* ==== Test Created with Cypress Studio ==== */
+it('can add a product into cart', function() {
+  /* ==== Generated with Cypress Studio ==== */
+  cy.visit('https://www.saucedemo.com/');
+  cy.get('[data-test="username"]').clear();
+  cy.get('[data-test="username"]').type('standard_user');
+  cy.get('[data-test="password"]').clear();
+  cy.get('[data-test="password"]').type('secret_sauce');
+  cy.get('[data-test="login-button"]').click();
+  cy.get('.title').should('be.visible');
```

Comment:
Can we modularize this logic? Meaning, can we wrap this code inside a Login method, and calling it in several places?

Reply:
@Prasanth-V-K sure, but this should be something that I will add soon we can't modularise this logic using Cypress Studio at the moment, makes sense? But it's on my radar to improve it, no worries

Reply:
Sounds good.